[IPL4893] unable to connect workspace to agent pool (#1119)
hashicorp/terraform-provider-tfe: CHANGELOG.md

<!-- Add CHANGELOG entry to this section for any PR awaiting the next release -->
<!-- Please also include if this is a Bug Fix, Enhancement, or Feature -->

FEATURES:

> * **New Resource**: `r/tfe_workspace__execution_mode` is a new resource to manage the execution mode on a workspace, by @laurenolivia [1119](https://github.com/hashicorp/terraform-provider-tfe/pull/1119)

Author: what about `manage` instead of `modify` since it's required? @brandonc

Author: It's also the language I use in our registry doc

> * **New Resource**: `r/tfe_workspace_agent_pool_execution` is a new resource to attach/detach an agent pool to a workspace, by @laurenolivia [1119](https://github.com/hashicorp/terraform-provider-tfe/pull/1119)

Reviewer: ```suggestion
* **New Resource**: `r/tfe_workspace_execution_mode` is a new resource to modify the execution mode of a workspace, by @laurenolivia [1119](https://github.com/hashicorp/terraform-provider-tfe/pull/1119)
```
